src/ft2font_wrapper.cpp

```diff
@@ -1668,9 +1676,19 @@ PYBIND11_MODULE(ft2font, m, py::mod_gil_not_used())
         .def_readonly("top", &PyPositionedBitmap::top)
         .def_property_readonly(
           "buffer", [](PyPositionedBitmap &self) -> py::array {
-            return {{self.bitmap.rows, self.bitmap.width},
+            if (self.bitmap.pixel_mode == FT_PIXEL_MODE_BGRA) {
+                return {
+                    py::array::ShapeContainer({self.bitmap.rows, self.bitmap.width, 4}),
```

Comment:
Not that it really matters but I think the explicit ShapeContainer and StridesContainer are optional?